butane/Cargo.toml

```diff
@@ -26,17 +26,14 @@ uuid = ["butane_core/uuid", "butane_codegen/uuid"]
 
 [dependencies]
 butane_codegen = { path = "../butane_codegen", version = "0.5" }
-butane_core = { path = "../butane_core", version = "0.5" }
+butane_core = { path = "../butane_core", version = "0.5", features = ["sqlite"] }
 
 
 [dev-dependencies]
 cfg-if = "1.0"
 exec_time = { version = "0.1.4" }
 paste = "1.0.11"
-chrono = { version = "0.4", default-features = false, features = [
```

Comment:
this was intentional ; see https://github.com/Electron100/butane/pull/47

Reply:
Reverted.